bot.rb

```diff
@@ -33,8 +33,16 @@ def start
     end
 
     def process_open_issue(issue)
+      bot_actions = []
       process_inactive(issue)
-      process_code_signing(issue)
+
+      return if issue.comments > 0 # there maybe already some bot replys
+      bot_actions << process_code_signing(issue)
+      bot_actions << process_env_check(issue)
+
+      bot_actions.each do |bot_reply|
+        client.add_comment(SLUG, bot_reply[:issue], bot_reply[:message])
```

Comment:
We don't need to also have the `issue` inside the hash, we already have access to it in the `issue` variable 👍